### PR TITLE
ASA-279: Some long overdue maintenance

### PR DESCRIPTION
**Pull request by bgrnwd**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @bgrnwd @tyler-richardett @doug-wade
```
Comment (tyler-richardett) [inline .github/CODEOWNERS:1]: image

¯\\\_(ツ)_/¯

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# itscalledsoccer
+# itscalledsoccer-js
 
 `itscalledsoccer` is a wrapper around the same API that powers the [American Soccer Analysis app](https://app.americansocceranalysis.com/). It enables Node users to programmatically retrieve advanced analytics for their favorite [MLS](https://en.wikipedia.org/wiki/Major_League_Soccer), [NWSL](https://en.wikipedia.org/wiki/National_Women%27s_Soccer_League), and [USL](https://en.wikipedia.org/wiki/United_Soccer_League) players and teams.
```
Comment (tyler-richardett) [inline README.md:3]: Just realized the R version has a [different summary](https://github.com/American-Soccer-Analysis/itscalledsoccer-r#overview) than the other two.

**Review by tyler-richardett — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (bgrnwd) [reply .github/CODEOWNERS:1]: I invited Doug to be a maintainer. Think he just needs to accept for that error to go away

File: README.md (hunk 2 above)
Comment (bgrnwd) [reply README.md:3]: I think that's alright

**Pull request merged**